marimo/_plugins/ui/_impl/input.py

```diff
@@ -1049,19 +1071,56 @@ def __init__(
         self,
         element: UIElement[JSONTypeBound, T],
         *,
+        bordered: bool = True,
+        loading: bool = False,
+        submit_button_label: str = "Submit",
+        submit_button_tooltip: Optional[str] = None,
+        submit_button_disabled: bool = False,
+        clear_on_submit: bool = True,
```

Comment:
Can this default to `False`? In the common case where the user is using `form` to gate submission of UI elements, it's helpful to see what was last submitted. The transition from `stale` to `submitted` already gives a visual cue to the user that the form was submitted.

```suggestion
        clear_on_submit: bool = False,
```